archive/frames/validators.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.serializers import ValidationError
+
+
+class ZipValidator():
+    def __call__(self, attrs):
+        selected_frames_count = len(attrs.get('frame_ids', []))
+        uncompress = attrs.get('uncompress', False)
+        if uncompress == True and selected_frames_count > 10:
```

Comment:
You could do `if uncompress and selected_frames_count > 10:`

Reply:
Lindy has ingrained in me that I'm never to use truthiness so as to avoid invoking his wrath, but I suppose it's coming through a ``BooleanField`` so there's no risk. As an aside I do believe Codacy dislikes that statement and prefers ``if uncompress is True``, so it was wrong in either case.